tagstudio/src/qt/widgets/thumb_renderer.py

```diff
@@ -764,6 +766,33 @@ def _video_thumb(self, filepath: Path) -> Image.Image:
             )
         return im
 
+    def _epub_cover(self, filepath: Path) -> Image.Image:
```

Comment:
it would be a nice to have test for similar this:

```python
from syrupy.extensions.image import PNGImageSnapshotExtension


def test_svg_to_png(snapshot):
    file_path = Path(__file__).parent / "fixtures" / "sample.epub"

    png_image = ThumbRenderer(...)._epub_cover(file_path, 600)
    assert png_image.read() == snapshot(extension_class=PNGImageSnapshotExtension)
```